src/test/java/com/wikia/webdriver/testcases/imageservingtests/ImageStorageTests.java

```diff
@@ -32,9 +32,9 @@
 
   @Test(groups = {"ImageStorageTests", "ImageStorage_001"})
   @UseUnstablePageLoadStrategy
-  public void ImageStorage_001_deleteImage_QAART_521() {
+  public void ImageStorage_001_deleteImage() {
     WikiBasePageObject base = new WikiBasePageObject(driver);
-    base.logInCookie(credentials.userNameStaff, credentials.passwordStaff, wikiURL);
+    base.logInCookie(credentials.userNameStaff2, credentials.passwordStaff2, wikiURL);
```

Comment:
I also saw an error on this test that when the test gets to the part where it needs to delete the file, the page shows that the user doesn't have permission. In fact, the log shows the user is no longer logged in. 
This is just an attempt to see if other tests are causing the user to get booted out of the session